airflow/providers/google/suite/transfers/sql_to_sheets.py

```diff
@@ -120,7 +120,7 @@ def execute(self, context: Any) -> None:
             impersonation_chain=self.impersonation_chain,
         )
 
-        self.log.info(f"Uploading data to https://docs.google.com/spreadsheets/d/{self.spreadsheet_id}")
+        self.log.info("Uploading data to https://docs.google.com/spreadsheets/d/%s", self.spreadsheet_id)
```

Comment:
This particular one is tricky; having a URL like this doesn’t really make sense without formatting. Maybe we should do

```suggestion
        if self.log.isEnabledFor("info"):
            url = f"https://docs.google.com/spreadsheets/d/{self.spreadsheet_id}"
            self.log.info("Uploading data to %s", url)
```

Reply:
+1

Reply:
Thanks for the pro tip!